experimental/dds/tree2/src/class-tree/internal.ts

```diff
@@ -22,5 +22,13 @@ export {
 	NodeBuilderData,
 } from "./schemaTypes";
 
-import * as testRecursiveDomain from "./testRecursiveDomain";
-export { testRecursiveDomain };
+// TODO:
+// Fix API-Extractor to support class based schema.
```

Comment:
Have we / can we file an issue with api-extractor about this? Any idea if the repro can be easily minimized?